Vic2ToHoI4/Source/HOI4World/HoI4Focus.h

```diff
@@ -43,6 +43,7 @@ class HoI4Focus: commonItems::parser
 
 		std::shared_ptr<HoI4Focus> makeCustomizedCopy(const std::string& country) const;
 		std::shared_ptr<HoI4Focus> makeTargetedCopy(const std::string& country, const std::string& target) const;
+		void updateFocusElement(std::string& element, std::string oldText, std::string newText);
```

Comment:
const std::string& for all constant inputs we're not modifying. Only place where const something& is not needed are primitives like int, double, bool etc.